Zeeguu Reader/Zeeguu Reader/ArticleView.swift

```diff
@@ -32,13 +32,14 @@ class ArticleView: UIView {
 	
 	var article: Article!
 	var titleLabel: UILabel?
-	var contentView: UITextView?
+	var contentView: ZGTextView?
+	var delegate: ArticleViewDelegate?
```

Comment:
also, what is the delegate? 
maybe pointer to docs if it's a well-known iOS GUI concept